meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh

```diff
@@ -531,6 +531,8 @@ fi
 # That's it if we only wanted supervisor update
```

Comment:
I certainly can expand on it, though it's your code that introduced this comment :) 
https://github.com/resin-os/meta-resin/commit/5dee4d12c0ac9d9d4f19e9769478a9695bd43f4a